tests/qos/test_buffer.py

```diff
@@ -1400,8 +1418,13 @@ def test_port_admin_down(duthosts, rand_one_dut_hostname, conn_graph_facts, port
     duthost = duthosts[rand_one_dut_hostname]
     original_speed = duthost.shell('redis-cli -n 4 hget "PORT|{}" speed'.format(port_to_test))['stdout']
     original_cable_len = duthost.shell('redis-cli -n 4 hget "CABLE_LENGTH|AZURE" {}'.format(port_to_test))['stdout']
-    original_profile = duthost.shell('redis-cli hget "BUFFER_PG_TABLE:{}:3-4" profile'.format(port_to_test))['stdout'][1:-1]
-    original_pg_size = duthost.shell('redis-cli hget "{}" size'.format(original_profile))['stdout']
+    if check_qos_db_fv_reference_with_table(duthost) == True:
+        original_profile = duthost.shell('redis-cli hget "BUFFER_PG_TABLE:{}:3-4" profile'.format(port_to_test))['stdout'][1:-1]
+        original_profile_key = original_profile
```

Comment:
The same. No need to add `original_profile_key`. just making `original_profile` prefixed by a table name should suffice.